src/main/java/teammates/common/util/Const.java

```diff
@@ -1250,6 +1251,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
                 + "try again in a few more minutes as the student may still be being saved.";
         
         public static final String STUDENT_EDITED = "The student has been edited successfully";
+        public static final String STUDENT_EDITED_AND_EMAIL_SENT = "The student has been edited successfully. "
```

Comment:
Make this `STUDENT_EDITED + " A summary ..."`? If you need the `STUDENT_EDIT` variable to contain period, go ahead and change it.